### DISCUSSION TITLE
Error in production [NEXTJS]

### DISCUSSION THREAD

**Discussion by sushanyadav** [vidstack/player#863] [Q&A]
```tsx
<MediaPlayer
        controls
        loop
        muted
        crossorigin="anonymous"
        datatype="video/mp4"
        src={[
          {
            src: `${ASSET_URL}/something/something.webm`,
            type: 'video/webm',
          },
          {
            src: `${ASSET_URL}/something/something.mp4`,
            type: 'video/mp4; codecs=hvc1',
          },
        ]}
      >
<MediaOutlet />
</MediaPlayer>
```

Error:

![CleanShot 2023-06-13 at 5 44 06](https://github.com/vidstack/player/assets/53527664/bb2b3d74-49c2-4ab4-9e0b-aeb451adea3d)

I am using react nextjs.

```
"@vidstack/react": "^0.6.10",
"vidstack": "^0.6.10"
```

![CleanShot 2023-06-13 at 5 46 44](https://github.com/vidstack/player/assets/53527664/831f7f91-b2dd-4656-830e-fa78eabf3ee8)

**Comment by mihar-22** [comment]
This has just been fixed, you'll need to destroy `node_modules` and your lockfile + re-install
